OntologyDecisionRecords/ODR 004 Introduction of Space:Wing.md

```diff
@@ -0,0 +1,15 @@
+# ODR 004 Introduction of Space:Wing
+
+## Context
+
+With the release of Brick 1.4 the brick:Location domain will be deprecated in favour of rec:Space. All specific Brick location classes but brick:Wing are already covered in REC. This ODR introduces the building Wing concept in REC for future compatibility reasons.
```

Comment:
The wording is hard to follow. Try:

With the release of Brick 1.4 the `brick:Location` domain will be deprecated in favour of `rec:Space`. All `Brick:location` classes are already covered in REC except for `brick:Wing`. This ODR introduces the Wing concept in REC for future compatibility reasons.